libs/cardano-protocol-tpraos/testlib/Test/Cardano/Protocol/Binary/Cddl.hs

```diff
@@ -0,0 +1,113 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -Wno-redundant-constraints #-}
+
+module Test.Cardano.Protocol.Binary.Cddl (
+  cddlEraSpec,
+  huddleEraSpec,
+  postBabbageCddlSpec,
+  postBabbageHuddleSpec,
```

Comment:
Doesn't "post"Babbage implies that it is applicable only after Babbage?
We could use `babbageOnwards`, but I think a better use the terminology that consensus uses?

```suggestion
  praosBlockCddlSpec,
  praosBlockHuddleSpec,
```

Reply:
I went back and forth about it. In the end i found some places in our code where "postBabbage" includes babbage, but I agree, it's ambiguous -  and it's ambiguous in the worst way possible - basically unclear about exactly what it wants to be precise about!!  
Thank you for the suggestion, I like it! :+1:

Reply:
> i found some places in our code where "postBabbage" includes babbage,

I would suggest avoiding using cardano-ledger-test as an example of good naming conventions :smiling_face_with_tear: